mix.exs

```diff
@@ -41,13 +41,15 @@ defmodule Community.Mixfile do
   defp deps do
     [
       {:cowboy, "~> 1.0"},
+      {:ecto, "~> 2.0.0-rc", [override: true, optional: false, hex: :ecto, manager: :mix]},
```

Comment:
This was recommended by `mix` because of conflicting dependencies among the packages.